doc/bc/changes-6.11.md

```diff
@@ -0,0 +1,16 @@
+# Backwards compatibility changes
+
+Changes affecting version compatibility with former or future versions.
+
+## Changes
+
+## Deprecations
+
+- EZP-26885: Field Type external storage
+  - Abstract base class `eZ\Publish\Core\FieldType\StorageGateway` for the gateway of a Field Type
```

Comment:
Yeah, sorry :( Somehow I thought we have only two deprecations in Core. I looked over entire diff and in fact we have 4 in total. Added as a fixup in 23722a4 for easier review.